src/jvm/main/org/jetbrains/kotlinx/lincheck/util/Spinner.kt

```diff
@@ -47,7 +47,7 @@ internal class Spinner(val nThreads: Int = -1) {
         var counter = 0
         val yieldLimit = 1 + if (shouldSpin) SPIN_CYCLES_BOUND else 0
         while (!condition()) {
-            Thread.onSpinWait()
+            threadOnSpinWait?.invoke(null)
```

Comment:
Invoking `Thread.onSpinWait` via reflection is not a good idea. Let's check how the performance changes with `Thread.onSpinWait` being omitted.

Reply:
I'll try method handles.

Reply:
I don't have ones in Java 8. Just check what changes without `Thread.onSpinWait`

Reply:
[It is even in Java 7](https://docs.oracle.com/javase/7/docs/api/java/lang/invoke/MethodHandle.html)

But I think we can just remove the method call. Is it OK?